packages/shared/src/variants/baduk_utils.ts

```diff
@@ -96,3 +97,16 @@ export function mapBoard<T, S>(
       return (board as T[][]).flat().map(f);
   }
 }
+
+export function equals_placement(
```

Comment:
Maybe a doc comment here because:

a) This function is doing a bit more than just equality check
b) We have a few terms around for similar concepts: "index", "coordinate", "intersection".  Would be good to explain what we mean by "placement".

Reply:
Added a doc comment.